framework/resource/retryresource/basic_resource.go

```diff
@@ -0,0 +1,116 @@
+package retryresource
```

Comment:
`basicResource` is a fallback resource when Resource is not an instance of `*framework.CRUDResource`.